docs/manifest_generation_cli.md

```diff
@@ -0,0 +1,25 @@
+## Manifest Generation CLI Usage
```

Comment:
Shouldn't we store this documentation in the CLI using the `help` argument? It would make it much easier to keep everything in sync.

Reply:
I should clarify that I'm referring to the options description. I think the example command should live in the main README since it's only one line.

Reply:
Thanks @BrunoGrandePhD. Yes, this makes sense, I've moved all the description(s) from the README files to dictionaries within the `help.py` module.